issuance/issuance.go

```diff
@@ -203,6 +192,27 @@ func NewProfile(profileConfig ProfileConfig, issuerConfig IssuerConfig) (*Profil
 	if issuerConfig.OCSPURL == "" {
 		return nil, errors.New("OCSP URL is required")
 	}
+
+	var policies []policyasn1.PolicyInformation
+	for _, policyConfig := range profileConfig.Policies {
+		id, err := parseOID(policyConfig.OID)
+		if err != nil {
+			return nil, fmt.Errorf("failed parsing policy OID %q: %s", policyConfig.OID, err)
+		}
+		policies = append(policies, policyasn1.PolicyInformation{Policy: id})
+	}
+	policyExtBytes, err := asn1.Marshal(policies)
+	if err != nil {
+		return nil, err
+	}
+	if len(policyExtBytes) == 0 {
+		return nil, errors.New("marshalled empty policy extension")
+	}
+	policyExt := &pkix.Extension{
```

Comment:
I think we don't need to manually encode this extension now that we are doing a plain list of OIDs. `x509.Certificate` has a `PolicyIdentifiers` field and `CreateCertificate` is [documented to use it](https://pkg.go.dev/crypto/x509#CreateCertificate).

Reply:
Aha! I'd totally forgotten that .PolicyIdentifiers was a thing! Done.